services/blockchain-connector/shared/sdk/fee.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const {
+	Logger,
+	Exceptions: { TimeoutException },
+} = require('lisk-service-framework');
+
+const { timeoutMessage, invokeEndpoint } = require('./client');
+
+const logger = Logger();
+
+let minFeePerByte;
+
+const getFeeConstants = async () => {
+	try {
+		// TODO: Update endpoint when available from SDK
+		const response = await invokeEndpoint('');
```

Comment:
Could you please finish the implementation according to the following file?
We can avoid future rework here :)
Let's have a offline discussion before you start making these changes.

https://github.com/LiskHQ/lisk-sdk/blob/e21e5d41d38df104c25632298f307ba8fb3fdf4f/framework/src/modules/fee/endpoint.ts.

Reply:
skipped as per offline discussion as this chages will be applied in https://github.com/LiskHQ/lisk-service/pull/1361